File: problemas/iniciante/2896/2896.ts
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = "";

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let lines = input.split('\n');
    let T = parseInt(lines.shift()!.trim());

    for (let i = 0; i < T; ++i){
        let [N, K]: number[] = lines.shift()!.trim().split(' ').map(Number);
        let garrafas: number = N;
        let vazias: number = N;

        while (vazias >= K) {
            let novas: number = Math.floor(garrafas / K);
            garrafas += novas;
            vazias = vazias % K + novas;
            break;
        }
        console.log(vazias);
    }
    
});

File: problemas/iniciante/2896/2896.py
def main():
    T = int(input())
    for _ in range(T):
        N, K = map(int, input().split())       
        garrafas = N
        vazias = N

        while vazias >= K:
            novas = garrafas // K
            garrafas += novas
            vazias = vazias % K + novas
            break
        
        print(vazias)

if __name__ == "__main__":
    main()
